samples/MerQure.Tools.Samples/RetryBusExample/Infra/SampleService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using MerQure.Tools.Buses;
+using MerQure.Tools.Configurations;
+using MerQure.Tools.Samples.RetryBusExample.Domain;
+using MerQure.Tools.Samples.RetryBusExample.Infra;
+using System;
+using System.Collections.Generic;
+
+
+namespace MerQure.Tools.Samples.RetryBusExample.RetryExchangeExample.Infra
+{
+	public class SampleService : ISampleService
+    {
+        public const string BusName = "sample";
+        private readonly IRetryBusService _retryBusService;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the field '_retryBusService' and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1450)